docs/user/alectryon.js

```diff
@@ -0,0 +1,190 @@
+var Alectryon;
```

Comment:
This file also comes from Alectryon right? There are no copyrights (that's surprising)?

Reply:
https://github.com/cpitclaudel/alectryon/blob/2a544230290be6dd84f0ce736e86f8a071d70739/alectryon/assets/alectryon.js#L4

Yep, but this is how it is upstream, I am fine with adding the compliance copyright, what do you prefer?

Reply:
(may be worth investigating https://reuse.software/ if we want to maximize the licensing compliance situation)